2DClothSim.h

```diff
@@ -5,5 +5,55 @@
 
 #include "SDL.h"
 #include <iostream>
+#include <vector>
+#include <math.h>
 
-// TODO: Reference additional headers your program requires here.
+namespace ClothSim {
+
+	SDL_Window* window;
+	SDL_Renderer* renderer;
+	double time;
+
+	struct vect2D {
+		float x;
+		float y;
+
+		vect2D(float inp_x, float inp_y) { x = inp_x; y = inp_y; }
+	};
+
+	class Cloth {
+
+	private:
+		int64_t pixels_x, pixels_y;
+		bool init;
+		std::vector<SDL_FPoint> *verts;
+		std::vector<struct vect2D> *velos, *accs;
+		float springConst, dampConst, vertMass;
+
+		const struct vect2D dirX = vect2D(1.0f, .0f), dirY = vect2D(.0f, 1.0f), dirNE = vect2D(sqrtf(0.5f), -sqrtf(0.5f)), dirSE = vect2D(sqrtf(0.5f), sqrtf(0.5f));
+
+		void initVerts();
+
+	public:
+		// Constructor
+		Cloth(int x = 100, int y = 100) : pixels_x(x), pixels_y(y), init(false), verts(nullptr), velos(nullptr), springConst(1.0f), dampConst(0.2f), vertMass(1.0f) { initVerts(); }
```

Comment:
didn't have a close look but 100 here and 10 at https://github.com/r41k0u/2DClothSim/pull/2/files#diff-18ad10c6fe7062603f10d7e8fa4f6b3dfa1e95feac928d9057ac4ca9f43364d4R48

Reply:
(10, 10) is the origin of the cloth, where the top left corner is. 2 is the distance between vertices